### ISSUE TITLE
Helping users sort existing CurseForge addons

### ISSUE THREAD

**Issue by kennywasafk** [AcidWeb/CurseBreaker#332]
Currently the way Cursebreaker just errors out when running across a curseforge addon makes it incredibly inconvenient and time consuming to migrate your addons. 

`RuntimeError: https://www.curseforge.com/wow/addons/bonusrollfilter
CurseForge is no longer supported by this application.
Please uninstall this addon and install it from different source.`

When running `status` this is what you get. It would be a great help if instead we could at least get the addon version to display, preferably within the familiar cursebreaker table alongside all the other addons. So instead of generating hundreds of lines of errors it would instead be neatly placed in the table just marked as "incompatible"

I understand this may seem like a not very high-priority change, but I think it would be very useful for people right now specifically, while everyone is migrating. 

Thanks in advance!

**Comment by AcidWeb** [comment]
I agree it should be displayed better. But due technical reasons it is surprisingly not so trivial. I will take another look.

**Comment by AcidWeb** [comment]
<img width="748" alt="image" src="https://user-images.githubusercontent.com/3713734/177367678-11d50263-0ab3-44f6-85cc-1d1f5b87122f.png">

**Comment by kennywasafk** [comment]
Looks brilliant, thanks for you time, will be looking forward to the release 👍

**Comment by Barleduq** [comment]
I opened cursebreaker about an hour ago, and it updated, yay!  However, I see the error lines mentioned in the starting comment, and not the nice display shown just before the ticket was closed. As that was 6 hours ago (according to github), I should have that version, no?  Or did it not get pushed to linux? Or do I need to manually download the new version, even tho it updated itself when I launched?
